aspnet/publishing/linuxproduction.rst

```diff
@@ -0,0 +1,190 @@
+Publish to a Linux Production Environment
+=========================================
+
+By `Sourabh Shirhatti`_
+
+In this guide, we will cover setting up a production-ready ASP.NET environment on an Ubuntu 14.04 Server. Though we will cover the specifics of using Ubuntu 14.04, the content will be applicable to other flavors of Linux.
+
+We will take an existing ASP.NET 5 application and place it behind a reverse-proxy server. We will then setup the reverse-proxy server to forward requests to our Kestrel web server.
+
+Additionally we will write a startup script to ensure our web application runs on startup as a daemon as well as a configure a process management tool to help restart our web application in the event of a crash to guarantee high availibility.
```

Comment:
"Though we will cover the specifics of using Ubuntu 14.04, the content will be applicable to other flavors of Linux."

Is this true? If the ICU dependency hasn't changed, then I question this statement.